packages/electrode-react-webapp/lib/express/index.js

```diff
@@ -5,17 +5,35 @@
 const _ = require("lodash");
 const assert = require("assert");
 const ReactWebapp = require("../react-webapp");
-
-const HTTP_REDIRECT = 302;
+const HttpStatus = require("http-status-codes");
 
 const handleRoute = (request, response, handler) => {
   return handler({ mode: request.query.__mode || "", request })
     .then(data => {
       const status = data.status;
+
+      // Status codes where we might want to keep custom html
+      const displayHtmlStatuses = [
+        HttpStatus.NOT_FOUND,
+        HttpStatus.GONE,
+        HttpStatus.SERVICE_UNAVAILABLE
+      ];
+
+      // Status codes where we want to redirect the user
+      const redirectStatuses = [
+        HttpStatus.MOVED_PERMANENTLY,
+        HttpStatus.MOVED_TEMPORARILY,
+        HttpStatus.PERMANENT_REDIRECT,
+        HttpStatus.TEMPORARY_REDIRECT
+      ];
+
       if (status === undefined) {
         response.send(data);
-      } else if (status === HTTP_REDIRECT) {
-        response.redirect(data.path);
+      } else if (redirectStatuses.find(redirectStatus => redirectStatus === status)) {
```

Comment:
why not use `indexOf`?

Reply:
Personal preference 😉 If you want, I can change this.